src/decorator-annotator.ts

```diff
@@ -197,7 +197,7 @@ export class DecoratorClassVisitor {
   }
 
   foundDecorators() {
```

Comment:
Maybe add a return type here?  From the function name it's not clear if it's returning a list of found decorators or if it's returning whether it found any.  (And it'd also have revealed this bug.)